src/Repository/ReaderRepository.php

```diff
@@ -1,19 +1,27 @@
 <?php
-/**
- * Created by PhpStorm.
- * User: ioana
- * Date: 11/28/2018
- * Time: 10:04 AM
- */
-include("DataBaseConnection.php");
 
+namespace BookReviews\Repository;
 
-class ReaderRepository extends DataBaseConnection
+/**
+ * Class ReaderRepository
+ * @package BookReview\Repository
+ */
+class ReaderRepository extends DatabaseConnection
 {
-     function InsertIntoDb($username, $password)
+    /**
+     * @param $username
+     * @param $password
+     */
+    public function insertIntoDatabase($username, $password)
     {
-
+         die();
+    }
+    /**
+     * @param string $username
+     * @param  string $password
```

Comment:
Param annotations should always be lined up on the type and on the variable, for example:
```
/**
* @param string             $var1
* @param DatabaseConnection $var2
*/
```